internal/pkg/composable/providers/kubernetes/README.md

```diff
@@ -1,20 +1,32 @@
-##  Autodiscover provider in Elastic Agent
+## Autodiscover provider in Elastic Agent
 
-https://www.elastic.co/guide/en/fleet/8.0/dynamic-input-configuration.html#dynamic-providers
+  
 
-Currently Kubernetes dynamic provider can only be configured in [standalone](https://github.com/elastic/beats/blob/03bf16907bea9768427f8305a5c345368b55d834/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L24) agent.
-In fleet managed agent it is enabled by default with default values.
+https://www.elastic.co/guide/en/fleet/current/kubernetes-provider.html
 
-Template based autodiscover of Kubernetes resources is only supported in standalone mode as of now.
-It is not part of the Kubernetes Integration yet.
+  
 
-Hints based autodiscover is not supported yet.
+Kubernetes provider can be configured both in **standalone** and **fleet managed** elastic agent.
 
-### Template based autodiscover
+a. In  **standalone** elastic agent, user needs to edit the [elastic-agent-standalone-kubernetes.yaml](https://github.com/elastic/elastic-agent/blob/f994f5bfdf68db27902a4175c3b655b4d611cf7c/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L28)
+b. In **fleet managed** elastic agent, kubernetes provider is enabled by default with default values. User needs to follow the steps described in the [documentation](https://www.elastic.co/guide/en/fleet/current/advanced-kubernetes-managed-by-fleet.html) to configure it.
+
+Condition based autodiscover is supported both in **fleet managed** elastic agent and in **standalone** elastic agent(see [doc](https://www.elastic.co/guide/en/fleet/current/conditions-based-autodiscover.html)).
+
+Hints based autodiscover is only supported in **standalone** elastic agent(see [doc](https://www.elastic.co/guide/en/fleet/current/hints-annotations-autodiscovery.html)).
+
+  
+
+### Conditions based autodiscover
+
+  
 
 Example:
-As an example we will use again redis module.
-In agent.yml(configmap) an extra input block needs to be added.
+
+As an example we will use the redis module.
+
+To automatically identify a Redis Pod and monitor it with the Redis integration, add the following input configuration inside the [Elastic Agent Standalone manifest](https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml):
```

Comment:
qq: for the Conditions-based autodiscover to work correctly is any kubernetes provider required to be enabled? If yes please provide the necessary settings of the provider in this example

Reply:
The default settings of the kubernetes provider should be enough for conditions to work.